.github/workflows/tests.yaml

```diff
@@ -51,12 +51,11 @@ jobs:
             ~/go/pkg/mod
             ~/go/bin
           key: unittest-${{ hashFiles('**/go.mod', '**/go.sum', '**/Makefile') }}
-      - name: Run unit tests
-        run: make test-coverage
+      - run: make test-coverage
```

Comment:
note: i figured the naming for the make target doesn't provide any helpful information so i rather have the raw make command exposed - that way anyone can see what they are supposed to do locally when this and elsewhere is failing